kiwi/builder/live.py

```diff
@@ -144,6 +144,8 @@ def create(self) -> Result:
                 'mbr_id': self.mbrid.get_id(),
                 'application_id': self.application_id,
                 'efi_mode': self.firmware.efi_mode(),
+                'efi_partition_table': self.firmware.get_partition_table_type(),
+                'gpt_hybrid_mbr': self.firmware.gpt_hybrid_mbr,
```

Comment:
@AdamWill can you add these settings also to `kiwi/builder/install.py` at self.custom_iso_args in the same way ? Otherwise it does not apply when building install ISO's for oem types. Thanks